apps/bfd-pipeline/bfd-pipeline-rif-load/src/main/java/gov/cms/bfd/pipeline/rif/load/RifLoader.java

```diff
@@ -174,7 +174,7 @@ public static EntityManagerFactory createEntityManagerFactory(DataSource jdbcDat
 
     Map<String, Object> hibernateProperties = new HashMap<>();
     hibernateProperties.put(org.hibernate.cfg.AvailableSettings.DATASOURCE, jdbcDataSource);
-    hibernateProperties.put(org.hibernate.cfg.AvailableSettings.HBM2DDL_AUTO, Action.VALIDATE);
+    hibernateProperties.put(org.hibernate.cfg.AvailableSettings.HBM2DDL_AUTO, Action.NONE);
```

Comment:
Won't having this same change give you a merge conflict? Or is Git / GitHub smart enough to avoid that?

Reply:
I believe GItHub will be smart enough after we rebase.